classes/class-ppc-pagesetups.php

```diff
@@ -52,11 +52,173 @@ public static function get_instance() {
+			$ppc_post_val_array       = isset( $ppc_total_checklist_data[ $current_screen_post_type->post_type ] ) ? $ppc_total_checklist_data[ $current_screen_post_type->post_type ] : '';
+			switch ( $columns ) {
+				case 'ppc_checklist':
+					if ( empty( $ppc_post_val_array ) ) {
+						echo esc_html( sprintf( __( 'No checklist found.', 'pre-publish-checklist' ) ) );
+					} elseif ( isset( $ppc_post_val_array ) && empty( $ppc_checked_data ) ) {
+						echo esc_html( sprintf( __( 'Checklist is empty', 'pre-publish-checklist' ) ) );
+					} elseif ( isset( $ppc_post_val_array ) && ! empty( $ppc_post_val_array ) && isset( $ppc_checked_data ) && ! empty( $ppc_checked_data ) ) {
+						$ppc_result = array_intersect_key( $ppc_post_val_array, $ppc_checked_data );
+						if ( isset( $ppc_result ) && ! empty( $ppc_result ) && count( $ppc_post_val_array ) > count( $ppc_result ) ) {
+							/* translators: %d: number term */
+							echo esc_html( sprintf( __( '%1$d items completed out of %2$d', 'pre-publish-checklist' ), count( $ppc_result ), count( $ppc_post_val_array ) ) );
+							echo '<br>';
+							?>
+						<progress value="<?php echo (int) count( $ppc_result ); ?>" max="<?php echo (int) count( $ppc_post_val_array ); ?>"></progress>
+							<?php
+						} elseif ( count( $ppc_post_val_array ) === count( $ppc_result ) ) {
+							echo esc_html( sprintf( __( 'Checklist is complete', 'pre-publish-checklist' ) ) );
+							echo '<br>';
+							?>
+						<progress value="<?php echo (int) count( $ppc_result ); ?>" max="<?php echo (int) count( $ppc_post_val_array ); ?>"></progress>
+							<?php
+						}
+					}
+					break;
+			}
+		}
+		/**
+		 * Function to add post column.
+		 *
+		 * @param string $columns for column.
+		 *
+		 * @since 1.1.0
+		 */
+		public function ppc_add_column_name_func( $columns ) {
+			$columns = array(
+				'cb'            => '<input type="checkbox"/)',
+				'title'         => 'Title',
+				'author'        => 'Author',
+				'categories'    => 'Categories',
+				'tags'          => 'Tags',
+				'comments'      => '<span class="vers comment-grey-bubble" title="Comments"><span class="screen-reader-text">Comments</span></span>',
+				'date'          => 'Date',
+				'ppc_checklist' => 'Checklist Status',
+			);
+			return $columns;
+		}
+		/**
+		 * Dropdown for page.
+		 *
+		 * @since 1.1.0
+		 */
+		public function add_dropdown() {
+			$ppc_pst_type       = get_option( 'ppc_post_types_to_display' );
+			$ppc_checklist_data = PPC_Loader::get_instance()->get_list();
+			if ( isset( $ppc_pst_type ) && isset( $ppc_checklist_data ) ) {
+				foreach ( $ppc_pst_type as $key ) {
+					$ppc_page_data     = isset( $ppc_checklist_data[ $key ] ) ? $ppc_checklist_data[ $key ] : '';
+					$current_post_type = get_current_screen();
+					if ( $current_post_type->post_type === $key ) {
+						?>
+						<select class="select_multiple" name="slect_opt" id="slect_opt">
+							<option value="">Filter Unchecked...</option>
```

Comment:
The text should be always ready to translate. here _Filter Unchecked_ is not translatable.

Reply:
<option value=""><?php esc_html_e( 'Filter Unchecked...', 'pre-publish-checklist' ); ?></option>

Reply:
Added <?php esc_html_e( 'Filter Unchecked...', 'pre-publish-checklist' ); ?>